simpledotorg/simple-server
# Restyle 'Drug stock report' and 'Drug stock report completed' views

**Story card:** [sc-6883](https://app.shortcut.com/simpledotorg/story/6883/update-drug-stock-report-view)

## Because

The "Drug stock report" form view and the "Drug stock report completed" view need to be updated to reflect the latest designs.

Note: The contents of the "Completed" view don't match the Figma file designs. The designs in Figma reflect the end state. This PR only restyles the page without moving the drug stock cards into the progress tab "Hypertension" report (which doesn't exist yet).

## This addresses

* Restyles the "Drug stock report" view
* Restyles the "Drug stock report completed" view

## Test instructions

1. Pull the latest, turn on your custom headers, and go to`http://localhost:3000/api/v3/analytics/user_analytics.html` to see the Progress Tab home page
2. Click the "Enter drug stock" button in the "Drug stock report" card
3. Check the new page—make sure editing the values in the drug stock report fields works and click send.
4. Check the new "completed" page—make sure the page looks good visually
